.travis.yml

```diff
@@ -1,61 +1,137 @@
+# Tell Travis to use new container-based infrastructure
 sudo: false
 
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
 language: php
 
-notifications:
-  email:
-    on_success: never
-    on_failure: change
-
+# Specify when Travis should build.
 branches:
   only:
     - master
 
+# Git clone depth.
+git:
+  depth: 1
+
 matrix:
   include:
-    - php: 5.3
-      env: WP_VERSION=latest
-      dist: precise
     - php: 5.6
-      env: WP_VERSION=latest
+      env: WP_VERSION=latest PHP_LINT=1
     - php: 7.0
-      env: WP_VERSION=latest
+      env: WP_VERSION=latest PHP_LINT=1
     - php: 7.1
-      env: WP_VERSION=latest
+      env: WP_VERSION=latest PHP_LINT=1
     - php: 7.2
-      env: WP_VERSION=latest
-    - php: 5.6
-      env: WP_VERSION=nightly
```

Comment:
Why remove the nightly version test?

Reply:
I didn't want the build to fail on anything other than a stable version, but I can look into adding this to `allow_failures` instead.

Reply:
I added this back, using `master` from the git repo, and added it to `allow_failures`.

Reply:
I'd recommend not allowing it to fail. I've never had a false positive from it, but it has warned me of legitimate issues in forthcoming releases. If Travis allows it to fail, nobody will ever notice when it does.

Reply:
OK, I'll remove the `allow_failures` block. Thanks!